src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -14,33 +18,43 @@ public class FlywheelSubsystem extends SubsystemBase {
      * Total: 2 TalonFX motors
      */
 
+    public TalonFX masterMotor;
+    public TalonFX followerMotor;
+
     public FlywheelSubsystem() {
+        masterMotor = new TalonFX(FlywheelConstants.LEFT_CAN_ID);
+        followerMotor = new TalonFX(FlywheelConstants.RIGHT_CAN_ID);
+
+        followerMotor.follow(masterMotor);
     }
 
     /*
      * Stop the flywheel from moving
      */
     public void stopFlywheel() {
+        this.masterMotor.neutralOutput();
+        this.followerMotor.neutralOutput();
```

Comment:
Since the follower motor will do the same thing as the master, you should never need to use `this.followerMotor` anywhere because it is redundant.

Reply:
got it.